.circleci/config.yml

```diff
@@ -105,4 +114,4 @@ workflows:
             tags:
               ignore: /addon-.*/
             branches:
-              only: master
+              only: main
```

Comment:
I'm not sure how this was working before this change, but I figured we should update this since we are using `main` as the default branch. Perhaps CircleCI aliases `master` with whatever your default branch is, but I haven't found any documentation to support that.